src/client/pages/ConnectPayment/components/TrustlyCheckout.tsx

```diff
@@ -45,21 +45,20 @@ export const TrustlyCheckout: React.FC<Props> = ({ onSuccess }) => {
     return res.data.registerDirectDebit.url
   }
 
+  const handleClick = async () => {
+    setTrustlyModalIsOpen(true)
+    const url = await generateTrustlyUrl()
+    if (!url) return
```

Comment:
```if (url) setTrustlyUrl(url)```

And how should we handle empty URL?

Reply:
Hmm that would have to be an error messages of some kind... "try again" -- let's ask design!

Reply:
This is the OLD solution tho so didn't plan on changing much here